src/mocks/users.mock.ts

```diff
@@ -1,7 +1,9 @@
 import UserProps from "../types/UserProps";
+import { v4 as uuidv4 } from 'uuid';
 
 const users: UserProps[] = [
     {
+        recipientUserId: uuidv4(),
```

Comment:
Why is there a recipientUserId and a id on a User?

Reply:
because someone called id, to the DNI field